main.py

```diff
@@ -2,6 +2,36 @@
 
 from django.db.models import QuerySet
 
+from db.models import Genre, Actor
+
 
 def main() -> QuerySet:
-    pass
+    # Create
+    genres = ["Western", "Action", "Dramma"]
+    for genre in genres:
+        Genre.objects.create(name=genre)
+
+    actors = [
+        ("George", "Klooney"),
+        ("Kianu", "Reaves"),
+        ("Scarlett", "Keegan"),
+        ("Will", "Smith"),
+        ("Jaden", "Smith"),
+        ("Scarlett", "Johanson"),
+    ]
+    for name, surname in actors:
+        Actor.objects.create(first_name=name, last_name=surname)
+
+    # Update
+    Genre.objects.filter(name="Dramma").update(name="Drama")
+    Actor.objects.filter(first_name="George", last_name="Klooney").\
+        update(last_name="Clooney")
+    Actor.objects.filter(first_name="Kianu", last_name="Reaves").\
```

Comment:
try to avoid using linebreakers here

Reply:
Here is the magic:
    Actor.objects.filter(
        first_name="Kianu",
        last_name="Reaves"
    ).update(first_name="Keanu", last_name="Reeves") ✨